composer.json

```diff
@@ -14,6 +15,7 @@
         }
     },
     "require": {
+        "php": "^7.2.24",
```

Comment:
Should we use `^` here or `>=`?

If we use `^` then when PHP 8.0 gets released we'll have to go through all our libraries and manually change that since `^7.2.24` is equivalent to `>=7.2.24 <8.0.0`. Would it be more future-proof to use `>=` instead? Or does it make more sense to use `^` since 8.0 will contain breaking changes and might break our library?

Just asking because I may update my PR on CIDR: https://github.com/dxw/cidr/pull/15

https://getcomposer.org/doc/articles/versions.md#caret-version-range-

Reply:
Good point.

I feel like using `>=` implies some sort of certainty that the library will work in PHP 8.0, whereas we don't know that will actually be the case.

Whilst the prospect of having to manually update composer files for any libraries where we've done something similar to `^7.2.24` doesn't particularly feel me with glee, I feel like that's probably preferable, as it should result in things failing during the build stage if we upgraded to 8.0, rather than the build succeeding but then the library erroring on staging/production.